### PR TITLE
chore: lab update

### PR DESCRIPTION
**Pull request by lukaisailovic**
# Description

Updated lab to include new branding, new links and featured set of configuration options

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

Screenshot 2024-09-20 at 15 37 04

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections
- [] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: apps/laboratory/src/utils/DataUtil.ts [hunk 1]
```diff
@@ -109,3 +116,60 @@ export const solanaSdkOptions = [
...
+    randomLinks: ['/library/wagmi-all-internal', '/library/ethers-all-internal']
+  }
+]
+
+export const featuredSdkOptions: SdkOption[] = [
+  {
+    title: 'Default',
+    link: '/library/wagmi',
+    description: 'Basic configuration using wagmi as a driving library'
+  },
```
Comment (tomiir) [inline apps/laboratory/src/utils/DataUtil.ts:164]: should we add ethers as well?

**Review by tomiir — APPROVED** [review]

**Comment by tomiir** [comment]
Are the sizes on the new mages ok? Seem small 🤔

File: apps/laboratory/src/utils/DataUtil.ts (hunk 1 above)
Comment (lukaisailovic) [reply apps/laboratory/src/utils/DataUtil.ts:164]: I don't think so, this is mostly for people showcasing our offerings. Using wagmi or ethers is irrelevant.

**Comment by lukaisailovic** [comment]
@tomiir I've noticed! They seem good res however. Will leave them.

**Pull request merged**